config/FeatureFlag.json

```diff
@@ -5,7 +5,7 @@
   "production_access": false,
   "quick_start": true,
   "switch_merchant": true,
-  "audit_trail": false,
+  "audit_trail": true,
```

Comment:
Do we need it to be true in all environments? And by default too?

Reply:
no, updated